ironfish-cli/src/utils/currency.ts

```diff
@@ -0,0 +1,80 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import {
+  displayIronAmountWithCurrency,
+  isValidAmount,
+  MINIMUM_IRON_AMOUNT,
+  RpcClient,
+  WebApi,
+} from '@ironfish/sdk'
+
+const REGISTER_URL = 'https://testnet.ironfish.network/signup'
+
+export async function verifyCanSend(
+  client: RpcClient,
+  api: WebApi,
+  flags: Record<string, unknown>,
```

Comment:
A good general tip is to abstract away CLI flags as soon as possible and instead use the values in the structure.

In this case, just having a param `expirationSequenceDelta` and passing that in would be both easier to read and more compartmentalized.

Reply:
yea agree, overlooked it.